assets/src/styles/editorStyle.scss

```diff
@@ -15,6 +15,8 @@
 @import "blocks/ArticlesEditor";
 @import "blocks/CarouselHeaderEditor";
```

Comment:
We can remove these as well, and re-add them when we merge wysiwyg beta (or not if we do code splitting)